src/ui/input/color-picker/color-picker.composition.tsx

```diff
@@ -5,34 +5,34 @@ import { ColorPicker, DEFAULT_COLOR } from './color-picker';
 export function PreviewColorPicker() {
   const [color, setColor] = useState<string>(DEFAULT_COLOR);
   return (
-    <ColorPicker value={color} onSelect={(value) => setColor(value.toString())} size="l" style={{ margin: 100 }}>
+    <ColorPicker value={color} onColorSelect={(value) => setColor(value)} size="l" style={{ margin: 100 }}>
       Select color
     </ColorPicker>
   );
 }
 
 export function DisabledColorPicker() {
   return (
-    <ColorPicker onSelect={() => {}} disabled size="l" value="#897DEC">
+    <ColorPicker onColorSelect={() => {}} disabled size="l" value="#897DEC">
```

Comment:
remove empty option (should be optional)

Reply:
(same for the others)

Reply:
done